src/extension/ios/iOSPlatform.ts

```diff
@@ -46,11 +44,13 @@ export class IOSPlatform extends GeneralMobilePlatform {
     private static readonly RUN_IOS_SUCCESS_PATTERNS = ["BUILD SUCCEEDED"];
 
     public showDevMenu(deviceId?: string): Q.Promise<void> {
-        return IOSPlatform.remote(this.runOptions.projectRoot).showDevMenu(deviceId);
+        // return IOSPlatform.remote(this.runOptions.projectRoot).showDevMenu(deviceId); // TODO replace with a new implementation from appLauncher
+        return Q.resolve(void 0);
```

Comment:
Let's try to avoid new usages of Q. It will help to handle #1113 faster in future.